conda/core/subdir_data.py

```diff
@@ -5,37 +5,33 @@
 import json
 import pickle
 import re
-import warnings
 from collections import UserList, defaultdict
 from contextlib import closing
-from errno import EACCES, ENODEV, EPERM, EROFS
+from errno import ENODEV
 from functools import partial
 from itertools import chain, islice
 from logging import getLogger
 from mmap import ACCESS_READ, mmap
-from os.path import dirname, exists, isdir, join, splitext
+from os.path import exists, join, splitext
 from pathlib import Path
 from time import time
 
 from genericpath import getmtime, isfile
 
-try:
-    from boltons.setutils import IndexedSet
-except ImportError:  # pragma: no cover
-    from .._vendor.boltons.setutils import IndexedSet
```

Comment:
We need to keep this import as-is, since `_vendor.boltons` has been deprecated (see [#12453](https://github.com/conda/conda/pull/12453/files#diff-ff8022ba5d08310acbbc6f95e1bc77a53f69e00ffe6daa51026af8880f89612dR24-R27))